match2/commons/match_group_display_helper.lua

```diff
@@ -1,9 +1,13 @@
+--
+-- @author QuadratClown for Liquipedia
+--
+
 local Array = require('Module:Array')
 local FnUtil = require('Module:FnUtil')
-local Json = require('Module:Json')
+local Json = require("Module:Json")
```

Comment:
```suggestion
local Json = require('Module:Json')
```

Ideally using a revert, applies everywhere in this file

Reply:
Thanks, looks like I started working from a not up to date version